react/javascript/src/components/customise/CustomRendering.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { messages } from '@cucumber/messages'
+
+export function mixinStyles(
+  builtIn: Record<string, string>,
+  custom?: Record<string, string>
+): Record<string, string> {
+  const mixed: Record<string, string> = {}
+  Object.keys(builtIn).forEach((key) => {
```

Comment:
Could simplify to use `Object.assign` if we aren't concerned with supporting IE.

Are we?

Reply:
I think that it could be managed by the transpiler(s) with some options?

In the code, I suggest we don't care, but we make sure we know how it looks like after transpilation.

Regarding the question regarding IE support, do we know how it is supported now? Does react - and other dependencies we rely on - actually still support it?

Reply:
Good point about transpilation. Because this is the `Object` API rather than a language/syntax feature, I wouldn't think transpilation would cover it, it would be more like a polyfill. I'll check it out though.

Reply:
In terms of React, implication is that IE11 is fine (for now) https://reactjs.org/docs/react-dom.html#browser-support.